src/content/docs/agents/index.mdx

```diff
@@ -1,18 +1,58 @@
 ---
 title: Agents
 template: basicDoc
+
```

Comment:
Is this extra line intentional?

Reply:
I wanted to dig into this a bit more. I don't know if this is from the `yaml.safeDump` or if its compiling that way from the MDAST. I'll take a look further and see if I can figure out how to remove this newline.

Reply:
Fixed in ae1eb4c